trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/home/component/ReelCard.kt

```diff
@@ -54,9 +54,11 @@ import org.jetbrains.compose.resources.stringResource
 @Composable
 internal fun FeedReelCard(
     reel: ReelUiState,
-    onLikeClick: () -> Unit,
-    onReelClick: () -> Unit
-) {
+    onClickLike: () -> Unit,
+    onClickReel: () -> Unit,
+    onExpandDescription: (String) -> Unit,
+
+    ) {
```

Comment:
we need a NICE `ctrl alt L` in here and delete the extra space in line 60 and the last comma in line 59 

```suggestion 
    onClickLike: () -> Unit,
    onClickReel: () -> Unit,
    onExpandDescription: (String) -> Unit
) {